conda_build/metadata.py

```diff
@@ -1994,6 +1997,14 @@ def get_rendered_output(self, name, permit_undefined_jinja=False):
                 break
         return output
 
+    @property
+    def force_ignore_keys(self):
+        return ensure_list(self.get_value('build/force_ignore_keys'))
+
+    @property
+    def force_use_keys(self):
+        return ensure_list(self.get_value('build/force_use_keys'))
```

Comment:
Sorry if this is a dump question, but could you please share an example of how these might be used?

Reply:
either of these are for when conda-build's detection of used variables isn't behaving.  It's a manual override.  If conda-build was looping over python when it should not have, then use force_ignore_keys to stop that looping.

Mostly I'm fed up with tweaking that detection code, so I'm throwing up my hands and saying people can deal with edge cases themselves.  If there are very common patterns that result in needing these often, then that's a good candidate for more time to get the detection to recognize the situation better.

Reply:
Thanks for the clarification and providing the option to override. Sounds like a good idea. :)